test/testcafe/spec/Registration_spec.js

```diff
@@ -7,7 +7,7 @@ async function setup(t) {
   const identityPage = new IdentityPageObject(t);
```

Comment:
why this test doesn't need mock?

Reply:
because on load we are always making the introspect call, which is what we will be mocking in this test as well.